keyboards/gmmk/pro/ansi/keymaps/gigahawk/keymap.c

```diff
@@ -0,0 +1,70 @@
+#include QMK_KEYBOARD_H
+
+#define _BASE 0
+#define _GAME 1
+#define _TRAN 2
```

Comment:
We highly recommend using enums for the layer names, rather than defines, as they are easier to manage long term.  
```suggestion
enum layers {
    _BASE,
    _GAME,
    _TRAN,
};
```